src/parser/parser.yy

```diff
@@ -907,11 +927,11 @@ edge_key_ref:
     ;
 
 fetch_edges_sentence
-    : KW_FETCH KW_PROP KW_ON name_label edge_keys yield_clause {
+    : KW_FETCH KW_PROP KW_ON fetch_labels edge_keys yield_clause {
         auto fetch = new FetchEdgesSentence($4, $5, $6);
         $$ = fetch;
     }
-    | KW_FETCH KW_PROP KW_ON name_label edge_key_ref yield_clause {
+    | KW_FETCH KW_PROP KW_ON fetch_labels edge_key_ref yield_clause {
```

Comment:
Emm, if don't implement it in this pr, please forbid this syntax at present.

Reply:
As the syntax has trouble to shift and reduce before edge_key and vids identifiers, I return the error msg in the executor.